### DISCUSSION TITLE
Any ideas on being able to give out XP?

### DISCUSSION THREAD

**Discussion by Tarang74** [danthe1st/Mee6-bypasser#33] [Ideas]
Will you be able to implement give-xp commands for mods?
Thanks

**Comment by danthe1st** [comment]
This is not the purpose of Mee6-bypasser. It does not provide another leveling system but it uses the one from Mee6 and automatically gives roles when it detects that someone reaches certain levels.

Giving out XP needs to be a feature of Mee6 in order to do that.
